tests/integration_tests/functional/test_balloon.py

```diff
@@ -50,6 +50,20 @@ def make_guest_dirty_memory(ssh_connection, should_oom=False, amount_mib=32):
         logger.error("stdout: %s", stdout)
         logger.error("stderr: %s", stderr)
 
+    # Aim OOM at fillmem process
+    cmd = "pidof fillmem"
+    pidof_exit_code, pidof_stdout, pidof_stderr = ssh_connection.run(cmd)
+    if pidof_exit_code != 0:
+        logger.error("while running: %s", cmd)
+        logger.error("stdout: %s", pidof_stdout)
+        logger.error("stderr: %s", pidof_stderr)
+    for pid in pidof_stdout.split(" "):
+        choom_exit_code, choom_stdout, choom_stderr = ssh_connection.run(f"choom -n 1000 -p {pid}")
+        if choom_exit_code != 0:
+            logger.error("while running: %s", cmd)
+            logger.error("stdout: %s", choom_stdout)
+            logger.error("stderr: %s", choom_stderr)
+
```

Comment:
I feel like between starting the fillmem process, and us doing this, the OOM killer will already have kicked in :/